autorag/evaluation/metric/generation.py

```diff
@@ -64,12 +64,25 @@ def compute_score(gt: List[str], pred: str) -> float:
     return result
 
 
-@generation_metric
-def bleu(gt: List[str], pred: str, **kwargs) -> float:
+#@generation_metric
```

Comment:
delete annotation of generation_metric decorator.
And please use `convert_inputs_to_list` decorator at here. It needs to be a python list for reducing error.